### ISSUE TITLE
RecursionError during tune_mr

### ISSUE THREAD

**Issue by prjemian** [APS-USAXS/ipython-usaxs#163]
this error was reported at **every** step of the tune:

```
Subscription value callback exception (EpicsSignalRO(read_pv='S:SRcurrentAI', name='aps_current', parent='aps', value=101.90035430331233, timestamp=1554310277.977068, auto_monitor=False, string=False))
Traceback (most recent call last):
  File "/APSshare/anaconda3/BlueSky/lib/python3.6/site-packages/ophyd/ophydobj.py", line 367, in inner
    cb(*args, **kwargs)
  File "/APSshare/anaconda3/BlueSky/lib/python3.6/site-packages/bluesky/run_engine.py", line 1576, in emit_event
    data, timestamps = _rearrange_into_parallel_dicts(obj.read())
  File "/APSshare/anaconda3/BlueSky/lib/python3.6/site-packages/ophyd/utils/epics_pvs.py", line 198, in wrapper
    return fcn(self, *args, **kwargs)
  File "/APSshare/anaconda3/BlueSky/lib/python3.6/site-packages/ophyd/signal.py", line 284, in read
    value = self.get()
  File "/APSshare/anaconda3/BlueSky/lib/python3.6/site-packages/ophyd/signal.py", line 828, in get
    self._read_changed(value=value, **info)
  File "/APSshare/anaconda3/BlueSky/lib/python3.6/site-packages/ophyd/signal.py", line 857, in _read_changed
    update=False, require_timestamp=True)
  File "/APSshare/anaconda3/BlueSky/lib/python3.6/site-packages/ophyd/signal.py", line 636, in _metadata_changed
    pvname, cl_metadata, require_timestamp=require_timestamp)
  File "/APSshare/anaconda3/BlueSky/lib/python3.6/site-packages/ophyd/signal.py", line 847, in _get_metadata_from_kwargs
    in self._metadata_key_map[pvname].items()
  File "/APSshare/anaconda3/BlueSky/lib/python3.6/site-packages/ophyd/signal.py", line 848, in <dictcomp>
    if cl_metadata.get(cl_key, None) is not None}
  File "/APSshare/anaconda3/BlueSky/lib/python3.6/site-packages/ophyd/signal.py", line 842, in fix_value
    if fixer_function is not None and value is not None
RecursionError: maximum recursion depth exceeded
```

**Comment by prjemian** [comment]
not a clue

**Comment by prjemian** [comment]
This was an ophyd problem from using a non-released GH master branch.  Should be fixed by using ophyd release 1.3.2 from conda.
